content/de/docs/tutorials/hello-minikube.md

```diff
@@ -75,7 +75,7 @@ Deployments sind die empfohlene Methode zum Verwalten der Erstellung und Skalier
 Der Pod führt einen Container basierend auf dem bereitgestellten Docker-Image aus.
 
     ```shell
-    kubectl create deployment hello-node --image=gcr.io/hello-minikube-zero-install/hello-node
+    kubectl create deployment hello-node --image=k8s.gcr.io/echoserver:1.10
```

Comment:
Can you make this echoserver:1.4 so it matches what is on https://minikube.sigs.k8s.io/docs/start/